components/ome-xml/pom.xml

```diff
@@ -46,6 +46,12 @@
     </dependency>
 
     <dependency>
+      <groupId>com.google.guava</groupId>
```

Comment:
@rleigh-dundee: the defaut merge command for the [breaking push job](https://ci.openmicroscopy.org/job/BIOFORMATS-DEV-breaking-push/) uses `merge -D none -I label:breaking -E label:exclude`. What you suggest should be achievable either by dropping `-D none` to include all organization PRs by default or by including specific PRs by number e.g  `merge -D none -I breaking -E exclude -I #xxx`.